spec/features/reviews/update_review_spec.rb

```diff
@@ -1,31 +1,57 @@
-# require 'rails_helper'
-#
-# feature 'user creates review', %Q{
-#   As an authenticated user
-#   I want to update an a review for an airline
-#   So that I can correct errors or provide new information
-# } do
-#   scenario 'user reviews updates sucessfully' do
-#     user = FactoryGirl.create(:user)
-#     FactoryGirl.create(:review)
-#     visit new_user_session_path
-#
-#     fill_in 'Email', with: user.email
-#     fill_in 'Password', with: user.password
-#
-#     click_button 'Log in'
-#
-#     visit airline_path(airline.id)
-#
-#     click_link('Edit')
-#
-#     select('1', from: 'Rating:')
-#     fill_in 'Body', with: 'I like this airline'
-#     click_button 'Edit review'
-#
-#     expect(page).to have_content('Review successfully edited')
-#     expect(page).to have_content("Rating: 1")
-#     expect(page).to have_content('I like this airline')
-#   end
-#
-# end
+require 'rails_helper'
+
+feature 'user updates review', %Q{
+  As an authenticated user
+  I want to update an a review for an airline
+  So that I can correct errors or provide new information
+} do
+  context 'as an authorized user' do
+    let(:user) { FactoryGirl.create(:user) }
+
+    before :each do
+      sign_in_as user
+    end
+
+    scenario 'user reviews updates sucessfully' do
+      airline = FactoryGirl.create(:airline)
+      review = FactoryGirl.create(:review, airline: airline, user: user)
```

Comment:
This doesn't need a variable name if you arn't referencing it ever again. Just factory create it so its in the DB